samples/app/src/main/java/com/example/app/token/Token.kt

```diff
@@ -43,11 +43,15 @@ fun Token(tokenViewModel: TokenViewModel) {
             horizontalArrangement = Arrangement.SpaceEvenly) {
             Button(
                 onClick = { tokenViewModel.getAccessToken() }) {
-                Text(text = "Refresh")
+                Text(text = "Get")
             }
             Button(
                 onClick = { tokenViewModel.forceRefresh() }) {
-                Text(text = "ForceRefresh")
```

Comment:
`ForceRefresh` button was good for testing. I see that now the 4 buttons don't fit in the UI, but can we possibly introduce a checkbox for the force refresh?

Reply:
![CleanShot 2024-06-19 at 10 06 50@2x](https://github.com/ForgeRock/forgerock-android-sdk/assets/18559472/babcb925-af73-4e77-8707-e079dff09db6)

Using short form, hope this fix all screen